API_DOC.md

```diff
@@ -193,3 +198,76 @@ params :verified      Boolean, default: false, desc: 'Whether the posting is ver
 ```
 api :DELETE, '/housings/:id'
 ```
+
+# Volunteer
+## Index
+```
+api :GET, '/volunteer'
+param :page     Integer, desc: "What page to get"
+param :per_page Integer, default: 25 desc: "How many results per page"
+```
+### Filters
+EX: `filter[FILTER_NAME]=FILTER_VALUE`
+
+Filterable Values:
+* `:city, type: :string`
```

Comment:
Seems like this is the housing#index endpoint, not volunteers

Reply:
Hehe you are right. The docs are wrong but the filters are right

Reply:
Filters meaning in the actual code